charts/SDFactory/values-cfx-int.yaml

```diff
@@ -0,0 +1,49 @@
+ingress:
```

Comment:
This repository is not the right place to place Cofinity environment specific values.yaml files.